### ISSUE TITLE
Powershell Script Fails to Import

### ISSUE THREAD

**Issue by Pentiva** [pxlrbt/move-wsl#16]
When I ran the powershell script it failed to import the tar file into WSL. 

I believe this line is incorrect:
https://github.com/pxlrbt/move-wsl/blob/bcedef62a8ac75068f9fd98f6731e8400e45226e/move-wsl.ps1#L86
It looks like the backticks and quotation marks are not matched correctly (and I'm not comfortable enough with powershell to figure out the correct matching).

**Comment by pxlrbt** [comment]
Hey,
did you get an error when importing the tar file? Did you have the rights to import at the location you specified?
I am not an PowerShell expert but the line looks fine. The same syntax is used for the export. The backticks are used for escaping the quotes.

**Comment by Pentiva** [comment]
Ah, I didn't realize that the backticks were escaping the quotes. The mismatch makes sense now.

Yes, the error was that the file couldn't be found (`The system cannot find the file specified`), but when I ran the command manually, it imported perfectly fine so I don't think it was a permission error. Unless it fails when ran from a different drive, because when manually running the command I ran it from the destination folder (`cmd /c wsl --import docker-desktop-data . docker-desktop-data.tar`)
